src/plugins/chart_expressions/expression_metric/public/components/metric_vis.tsx

```diff
@@ -92,6 +94,10 @@ const getMetricFormatter = (
     );
   }
 
+  if (formatId === 'string' || 'null') {
```

Comment:
Seems like this will always evaluate to true because it will be `true` or `'null'` (a truthy string). Did you mean

```suggestion
  if ((formatId === 'string') || (formatId === 'null')) {
```

Reply:
It is not ready to be reviewed. I know about it. Still on the draft mode!

Reply:
Sorry, @stratoula ! My mistake.